lib/DDG/Spice/TodayInHistory.pm

```diff
@@ -3,33 +3,83 @@ package DDG::Spice::TodayInHistory;
 
 use strict;
 use DDG::Spice;
+use DateTime;
+use DateTime::TimeZone;
+use Time::Piece;
 
-name 'Today In History';
-description 'this day in history';
-source 'History.com';
-primary_example_queries 'today in history';
-secondary_example_queries 'this day in history';
-category 'facts';
-topics 'everyday', 'special_interest';
-icon_url 'http://www.history.com/favicon.ico';
-code_url 'https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/lib/DDG/Spice/TodayInHistory.pm';
-attribution web => [ 'https://www.duckduckgo.com', 'DuckDuckGo' ],
-            github => [ 'https://github.com/duckduckgo', 'duckduckgo'],
-            twitter => ['http://twitter.com/duckduckgo', 'duckduckgo'];
-
-triggers startend => (
-    'today in history',
-    'todays in history',
-    'this day in history',
-);
-
-spice to => 'http://www.history.com/this-day-in-history/rss';
-
-spice wrap_string_callback => 1;
+name "TodayInHistory";
+source "http://wikipedia.com";
+description "List of events which occured on this day in history";
+primary_example_queries "today in history", "this day in history";
+secondary_example_queries "historical events on 27 June";
+category "dates";
+topics "everyday","trivia";
+code_url "https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/lib/DDG/Spice/TodayInHistory.pm";
+attribution github => ["https://github.com/killerfish", "Usman Raza"],
+            twitter => ["https://twitter.com/f1shie", "Usman Raza"];
+
+spice to => 'http://en.wikipedia.org/w/api.php?action=query&titles=$1&prop=revisions&rvlimit=1&rvprop=content&format=json&continue=&redirects&callback={{callback}}';
```

Comment:
Wikipedia now redirects all HTTP to HTTPS. We can't have any redirects in the Spice endpoint. Please change "http://" to "https://"